precompile/contracts/juror/logic.go

```diff
@@ -386,6 +387,114 @@ func validateExecuteIOCOrder(bibliophile b.BibliophileClient, order *orderbook.I
 	}, nil
 }
 
+func GetPrevTick(bibliophile b.BibliophileClient, input GetPrevTickInput) (*big.Int, error) {
+	if input.Tick.Sign() == 0 {
+		return nil, errors.New("tick price cannot be zero")
+	}
+	if input.IsBid {
+		bidsHead := bibliophile.GetBidsHead(input.Amm)
+		if input.Tick.Cmp(bidsHead) != -1 {
+			return nil, fmt.Errorf("tick %d is greater than or equal to bidsHead %d", input.Tick, bidsHead)
+		}
```

Comment:
We dont need this check. It is already covered in contract and logic will return bidsHead if called anyway.
Same for asks